gym/lib/gym/generators/package_command_generator_xcode7.rb

```diff
@@ -28,6 +28,10 @@ def options
         options << "-archivePath '#{BuildCommandGenerator.archive_path}'"
         options << "-exportPath '#{temporary_output_path}'"
 
+        if Gym.config[:toolchain]
+          options << "-toolchain '#{Gym.config[:toolchain]}'"
```

Comment:
So this value gets passed to both `xcodebuild` commands, for building and for archiving?